webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,11 +7,22 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <base-button
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:40:58Z" title="Thursday, January 16th 2020, 3:40:58 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,7 +7,14 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <ds-button
```


```suggestion
      <base-button
```

please refactor to use only our new `base-button`